drone-webdriver/src/main/java/org/jboss/arquillian/drone/webdriver/binary/handler/AbstractBinaryHandler.java

```diff
@@ -185,7 +185,7 @@ protected File downloadAndPrepare(File targetDir, URL from) throws Exception {
         File downloaded = Downloader.download(targetDir, from);
         File extraction = BinaryFilesUtils.extract(downloaded);
         File[] files = extraction.listFiles(file -> file.isFile());
```

Comment:
Yes. Selenium 3.x.z is only JDK8 compatible. 
I decided to do it when I released the first alpha version.